ui/app/src/components/Dialogs/PathSelectionDialog.tsx

```diff
@@ -261,7 +261,8 @@ function PathSelectionDialogWrapper(props: PathSelectionDialogProps) {
           {formatMessage(messages.ok)}
         </Button>
       </DialogFooter>
-      <CreateNewFolderDialog
+      {/* TODO: use Dispatch*/}
```

Comment:
ToDo

Reply:
cant use dispatch because this is used on siteConfig